### ISSUE TITLE
Can we add extra label like "skip" on the Showcase

### ISSUE THREAD

**Issue by Saudasadaf** [amlcurran/ShowcaseView#366]

**Comment by amlcurran** [comment]
As in, you would like to add another button? No that's not currently possible, but certainly something I would consider adding

**Comment by Saudasadaf** [comment]
yes, like a extra button. we can hide it when ever required right?

**Comment by kevindekemele** [comment]
As I need this as well, I am currently implementing this locally (will do pull request if finished)
It's just an extra button which has similar methods as the first button.

However I have a question about design,
Currently, I made a method "Builder addButtonTwo()'' , after which you can call "Builder replaceButtonTwo()'' 

If you do it the other way around, it would not work, as I require the user to call addButtonTwo first.
Is this good design or should I change this?

**Comment by amlcurran** [comment]
@kevindekemele I'm not sure about the design of that you're suggesting, it'd be better if there is only one method to add a button. I'll have a think if there is a better way of doing this but otherwise that is a good plan

**Comment by kevindekemele** [comment]
I've added a pull request. @Saudasadaf you can program the button any way you want, so 'skip' as well.

**Comment by Saudasadaf** [comment]
Thanks, I have already implemented that locally.

**Comment by amlcurran** [comment]
So I had a go an doing this and it is actually possible to implement without changing the ShowcaseView API itself. You can do this:

```java
Button button = (Button) LayoutInflater.from(this).inflate(R.layout.view_custom_button, showcaseView, false);
RelativeLayout.LayoutParams secondParams = new RelativeLayout.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT);
secondParams.addRule(RelativeLayout.ALIGN_PARENT_BOTTOM);
secondParams.addRule(RelativeLayout.ALIGN_PARENT_RIGHT);
secondParams.setMargins(margin, margin, margin, margin);
showcaseView.addView(button, secondParams);
```

If you do this, can you do everything you need? Because then we don't have to change the API :)
